### PR TITLE
Add PIP project to team config

### PR DESCRIPTION
**Pull request by johnoneill46**

### CODE REVIEW

**Review by kurt-uk — APPROVED** [review]
well done

File: team-config.yml [hunk 1]
```diff
@@ -30,3 +30,11 @@ apple-shared-infrastructure:
...
   tags:
     application: core
+pip:
+  team: "Publishing Information Project"
+  namespace: "pip"
+  slack:
+    contact_channel: "#publishing-and-information"
+    build_notices_channel: "#pip-build-notices"
+  tags:
+    application: core
```
Comment (adusumillipraveen) [inline team-config.yml:40]: these are used to evaluate cost centre, should probably be more relevant.

File: team-config.yml [hunk 2]
```diff
@@ -31,10 +31,10 @@ apple-shared-infrastructure:
...
 pip:
-  team: "Publishing Information Project"
+  team: "Future Hearings - Publishing Information Project"
   namespace: "pip"
   slack:
     contact_channel: "#publishing-and-information"
     build_notices_channel: "#pip-build-notices"
   tags:
-    application: core
+    application: pip
```
Comment (adusumillipraveen) [inline team-config.yml:40]:
```suggestion
    application: publishing-information
```

Comment (adusumillipraveen) [reply team-config.yml:40]: pip could be confusing sometimes with infra terminology (public ip address)

**Review by adusumillipraveen — APPROVED** [review]

**Pull request merged**